test/basic.html

```diff
@@ -78,9 +85,8 @@
         test('set opened to true', function(done) {
           collapse.opened = true;
```

Comment:
at this point collapse is already opened, so this test is not executed in the right conditions. We should set it first to false, then see if it manages to switch to true and set the right height